src/sentry/api/bases/organization_events.py

```diff
@@ -76,6 +76,22 @@ def get_snuba_query_args(self, request, organization):
         return snuba_args
 
     def get_snuba_query_args_v2(self, request, organization, params):
+        group_ids = request.GET.getlist('group')
+        if group_ids:
+            try:
+                group_ids = set(map(int, filter(None, group_ids)))
+            except ValueError:
+                raise OrganizationEventsError('Invalid group parameter. Values must be numbers')
+
+            projects = Project.objects.filter(
+                organization=organization,
+                group__id__in=group_ids,
+            ).distinct()
+            if any(p for p in projects if not request.access.has_project_access(p)):
+                raise PermissionDenied
+            params['issue.id'] = list(group_ids)
+            params['project_id'] = list(set([p.id for p in projects] + params['project_id']))
```

Comment:
Should we make a common method with the implementation in `get_snuba_query_args`?

Reply:
I think we can just leave this here since
- this behavior is really specific to this endpoint
- we'll deprecate the original version when this is released, so the duplication would be short lived
- i'm not sure whether this behavior is going to stick around (i.e. when this becomes supported by OR search)